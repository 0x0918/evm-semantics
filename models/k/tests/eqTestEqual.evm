PUSH 3 ; PUSH 3 ; EQ; skip
