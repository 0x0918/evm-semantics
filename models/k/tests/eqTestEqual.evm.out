<T> <k> . </k> <accountID> 1 </accountID> <pc> 3 </pc> <wordStack> 1 : .WordStack </wordStack> <localMem> .Map </localMem> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 3 ) 1 |-> ( PUSH 3 ) 2 |-> EQ </program> <storage> .Map </storage> <balance> 40 </balance> </account> </accounts> </T>
